rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/v1/SwaggerDefinitionConfig.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2017 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.rest.service.v1;
+
+import io.swagger.annotations.Contact;
+import io.swagger.annotations.ExternalDocs;
+import io.swagger.annotations.Info;
+import io.swagger.annotations.License;
+import io.swagger.annotations.SwaggerDefinition;
+
+@SwaggerDefinition(
+        info = @Info(
+                description = "This is an example REST service for Gaffer.",
```

Comment:
Please could this just say "The Gaffer REST service"  or perhaps we don't need the description.